src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -87,6 +87,14 @@ class PyGameObject:
     Graphic: int = None
     Hue: int = None
 
+    def HasLineOfSightFrom(observer: Any = None) -> bool:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)